pkg/policy/selectorcache.go

```diff
@@ -315,7 +459,8 @@ func (sc *SelectorCache) UpdateIdentities(added, deleted cache.IdentityCache) {
 				idSel.notifyUsers(adds, dels)
 			}
 		case *fqdnSelector:
-			// TODO
+			// This is a no-op right now. We don't encode in the identities
+			// which FQDNs they correspond to.
```

Comment:
It may be problematic to try to include this anyway. The DNSName -> IP may be a many to one mapping, and it will change over time but the identity itself won't. This might also conflict with an IP when it is directly added via toCIDRs. The selector -> IP mapping might have a larger cardinality too, since `MatchPattern: "*"` is a common rule for L7, and could be for L3 too.